docusaurus.config.js

```diff
@@ -139,6 +139,20 @@ const config = {
   ],
 
   plugins: [
+    [
+      '@docusaurus/plugin-content-docs',
```

Comment:
Nice, this looks like a better version of the custom one we wrote `externalContentPlugin`.

Reply:
If it does the same job, I vote yes! If it's too time consuming though, you can open an issue or do it in a separate PR.